modules/core/arrow-core/src/main/kotlin/arrow/core/Try.kt

```diff
@@ -123,16 +123,19 @@ sealed class Try<out A> : TryOf<A> {
     }
   }
 
-  fun toOption(): Option<A> = fold({ None }, { Some(it) })
+  fun toOption(): Option<A> = fold({ None }, {
+    arrayListOf<String>().fold
```

Comment:
What's this about?

Reply:
Euh, nothing :)
![image](https://media.giphy.com/media/l2JJKs3I69qfaQleE/giphy.gif)